ccp/modules/exporter-setup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
+
+if [ "$ENABLE_EXPORTER" == true ]; then
```

Comment:
Please add OR "$ENABLE_DATASHIELD" == true

Reply:
something like this should work: `if [[ "$ENABLE_EXPORTER" == "true" || "$ENABLE_DATASHIELD" == "true" ]]; then echo "true"; else echo "false";fi`